atlasdb-config/src/main/java/com/palantir/atlasdb/factory/timelock/BlockingSensitiveConjureTimelockService.java

```diff
@@ -37,12 +37,12 @@
  * Given two proxies to the same set of underlying TimeLock servers, one configured to expect longer-running operations
  * on the server and one configured not to, routes calls appropriately.
  */
-public final class BlockingSensitiveConjureTimelockService implements ConjureTimelockService {
-    private final ConjureTimelockService blocking;
-    private final ConjureTimelockService nonBlocking;
+public final class BlockingSensitiveConjureTimelockService implements ConjureTimelockServiceBlocking {
+    private final ConjureTimelockServiceBlocking blocking;
+    private final ConjureTimelockServiceBlocking nonBlocking;
```

Comment:
this type & variable name combo is fantastically misleading... IIRC the two fields have different timeouts?

Reply:
Macro vs micro blocking :)